Assets/MixedRealityToolkit/_Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs

```diff
@@ -77,6 +88,16 @@ public class MixedRealityBoundaryVisualizationProfile : BaseMixedRealityProfile
         /// </summary>
         public Material PlayAreaMaterial => playAreaMaterial;
 
+        [PhysicsLayer]
+        [SerializeField]
+        [Tooltip("The physics layer to assign to the generated play area.")]
+        private int playAreaPhysicsLayer = 2;
```

Comment:
```suggestion
        private int playAreaPhysicsLayer = 0; // Allow physics interactions with the play area plane
```

Reply:
Play area was on the ignore layer previously. [Let's keep it the same as it was before](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3251/files#r240363203).

Reply:
fair enough. makes sense since it sits atop the floor